snapraid-runner.py

```diff
@@ -9,9 +10,11 @@
 import threading
 import time
 import traceback
+import json
+import requests
 from collections import Counter, defaultdict
 from cStringIO import StringIO
-
+from slackclient import SlackClient
```

Comment:
No need really to add this dependency for everyone - could instead make it optional and import it within `send_slack`? (cf.`send_email`)

Reply:
Good point, I don't use the slack client with my other code at this point. I use requests to make slack calls